### PR TITLE
Bugfix/service logo field mime type issue

### PR DESCRIPTION
**Pull request by appsol**
### Summary
https://trello.com/c/o72VuZiY

The user had a number of issues around creating organisations. The issues related to a lack of clear error messages, or error messages not showing at all.
The image upload component will now state clearly if an image has an unacceptable mime type.
The social media component will now show errors if the type field is not selected. The provided error is not particularly clear, but this can be addressed better in the api.

### Development checklist
- [x] The code has been linted `npm run lint --fix`

### Release checklist
NA

### Notes
NA

### CODE REVIEW

**Comment by matthew-inamdar** [comment]
@appsol could you take a look at the merge conflicts please 🙂

**Comment by appsol** [comment]
Resolved Merge conflicts

File: src/views/organisations/Index.vue [hunk 1]
```diff
@@ -301,8 +301,6 @@ export default {
         })
       });
 
-      this.$refs.organisationsTable.fetchResources();
```
Comment (matthew-inamdar) [inline src/views/organisations/Index.vue:304]: Is this supposed to be removed?

Comment (appsol) [reply src/views/organisations/Index.vue]: Not 100% sure. I ran the branch and it was displaying some bugs which were fixed in develop, so I've merged in develop and retained that line and everything seems to be OK now.

**Review by matthew-inamdar — APPROVED** [review]
LGTM 👍

**Pull request merged**